http-client/src/main/java/com/artipie/http/client/jetty/JettyClientSlice.java

```diff
@@ -80,9 +84,9 @@ final class JettyClientSlice implements Slice {
 
     @Override
     public Response response(
-        final String line,
-        final Iterable<Map.Entry<String, String>> headers,
-        final Publisher<ByteBuffer> body
+            final String line,
+            final Iterable<Map.Entry<String, String>> headers,
+            final Publisher<ByteBuffer> body
```

Comment:
I see in changes extra spacing, 4 to 8 spaces, here and several places below and above. I think 4 spaces is nice indentation, since it more compact than 8, but still distinctive.